cmd/generic-handlers.go

```diff
@@ -436,55 +436,6 @@ func setBucketForwardingHandler(h http.Handler) http.Handler {
 			return
 		}
 
-		// For browser requests, when federation is setup we need to
-		// specifically handle download and upload for browser requests.
-		if guessIsBrowserReq(r) {
-			var bucket, _ string
-			switch r.Method {
-			case http.MethodPut:
```

Comment:
This is removed since we do not have an embedded browser anymore and console UI handles federation by regular S3 calls so no special treatment is necessary.